concent_api/core/message_handlers.py

```diff
@@ -464,6 +467,9 @@ def handle_send_force_subtask_results(client_message: message.concents.ForceSubt
 
     current_time = get_current_utc_timestamp()
 
+    if Subtask.objects.filter(subtask_id=task_to_compute.compute_task_def['subtask_id']).exists():  # pylint: disable=no-member
+        Subtask.objects.select_for_update().get(subtask_id=task_to_compute.compute_task_def['subtask_id'])  # pylint: disable=no-member
+
     if Subtask.objects.filter(  # pylint: disable=no-member
```

Comment:
These three queries can be replaced with one.

Reply:
done